microsoft-edge/webview2/release-notes.md

```diff
@@ -104,6 +104,78 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 <!-- end of top matter -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2045.28
+
+Release Date: September 18, 2023<!-- todo: finalize -->
+
+[NuGet package for WebView2 SDK 1.0.2045.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2045.28)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 117.0.2045.28 or higher.
+
+#### General

```

Comment:
```suggestion
#### All platforms
```
design proposal idea.  When there's a General heading but no .NET or WinRT or Win32/C++ heading at same level like in early releases, the meaning of "General" is lost.  It really means "All platforms", I think.

Reply:
pushed commit for that design